projects/models.nix

```diff
@@ -24,6 +24,12 @@ let
     any
     ;
 
+  subgrantType = struct "subgrant" {
```

Comment:
Should we add `Commons` as well? It's missing from the GitHub labels.

Reply:
Yes!